src/components/style.css

```diff
@@ -0,0 +1,81 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    list-style: none;
+    font-family: Georgia, 'Times New Roman', Times, serif;
+}
+input:focus{
+    outline: none;
+}
+
+#topDiv{
+    border: 1px solid grey;
+    border-radius: 5px;
+    width: 80%;
+    margin-left: 10%;
+    margin-top: 2%;
+
+}
+#buttomDiv{
+    border: 1px solid #fff;
+    border-radius: 5px;
+    width: 80%;
+    margin-left: 10%;
+    margin-top: 4%;
+}
+
+#headInput{
+    font-weight: bold;
+    font-size: 18px;
+    width: 98%;
+    border: 1px solid #fff;
+    margin-top: 10px;
+    margin-left: 1%;
+    padding: 10px;   
+}
+#dutyInput{
+    width: 98%;
+    border: 1px solid #fff;
+    margin-top: 10px; 
+    margin-left: 1%;
+    padding: 10px;  
+}
+
+select, button{
```

Comment:
select, button, li gibi aslında sayfadaki belli bir bölümü düzenlemek için yazdığın css'leri bu şekilde direk element lere yazarsan ileride sorun yaşarsın. O yüzden class lar kullanıp etki alanını daraltmalısın.